utils/check_cluster_upgrade.sh

```
@@ -16,8 +16,15 @@ usage() {
 
 cleanup() {
   echo "-- Cleaning up --"
-  echo "Stopping postmaster on ${DATADIR} up"
-  ${BIN_OLD}/pg_ctl -D ${DATADIR} stop
+  if test -f "${DATADIR}/postmaster.pid"; then
+    if test -n "${BIN_NEW}"; then
+      echo "Stopping postmaster on ${DATADIR}"
+      ${BIN_NEW}/pg_ctl -D ${DATADIR} stop
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)